src/coldfront_plugin_api/scim_v2/auth_middleware.py

```diff
@@ -0,0 +1,28 @@
+import logging
+
+from django_scim.middleware import SCIMAuthCheckMiddleware
+from mozilla_django_oidc.contrib.drf import OIDCAuthentication
+
+logger = logging.getLogger(__name__)
+
+
+class SCIMColdfrontAuthCheckMiddleware(SCIMAuthCheckMiddleware):
+    def __call__(self, request, *args, **kwargs):
```

Comment:
This seems identical to the one in the superclass, why did you redefine it?

Reply:
Sorry. This was oversight by me